### DISCUSSION TITLE
How to translate the slider with loop: true

### DISCUSSION THREAD

**Discussion by thibka** [nolimits4web/swiper#7441] [Q&A]
Hi,

Slides are not looping when using the `translateTo()` method (see [demo](https://codesandbox.io/p/sandbox/swiper-translateto-loop-mch7w6)).
I've tried many different things (`slideToLoop()`, `setTranslate()`, `setProgress()`, ...), but can't find a way to programmatically translate the slider while keeping the loop working. 
Is that possible? 
I'm open to a different way, my end goal is to simply be able to freely animate the slides.

Thanks!

**Comment by elia-orsini** [comment]
@thibka Did you ever find a way?

**Reply by thibka** [reply]
@elia-orsini Can't remember for sure, but I think I ended up using Keen Slider.

**Reply by elia-orsini** [reply]
Oh, that's a shame! I will open a new issue maybe
